Formula/r.rb

```diff
@@ -0,0 +1,172 @@
+    else
+      args << "--with-aqua"
+    end
+
+    if build.with? "openblas"
+      args << "--with-blas=-L#{Formula["openblas"].opt_lib} -lopenblas" << "--with-lapack"
+      ENV.append "LDFLAGS", "-L#{Formula["openblas"].opt_lib}"
+    else
+      args << "--with-blas=-framework Accelerate" << "--with-lapack"
+      ENV.append_to_cflags "-D__ACCELERATE__" if ENV.compiler != :clang
+    end
+
+    # Help CRAN packages find gettext and readline
+    %w[gettext readline].each do |f|
+      ENV.append "CPPFLAGS", "-I#{Formula[f].opt_include}"
+      ENV.append "LDFLAGS", "-L#{Formula[f].opt_lib}"
+    end
+
+    # Pull down recommended packages if building from HEAD.
+    system "./tools/rsync-recommended" if build.head?
+
+    system "./configure", *args
+
+    system "make"
+    ENV.deparallelize # Serialized installs, please
+    system "make", "install"
+
+    # Link binaries, headers, libraries, & manpages from the Framework
+    # into the normal locations
+    bin.install_symlink prefix/"R.framework/Resources/bin/R"
+    bin.install_symlink prefix/"R.framework/Resources/bin/Rscript"
+    frameworks.install_symlink prefix/"R.framework"
+    include.install_symlink Dir[prefix/"R.framework/Resources/include/*"]
+    lib.install_symlink prefix/"R.framework/Resources/lib/libR.dylib"
+    man1.install_symlink prefix/"R.framework/Resources/man1/R.1"
+    man1.install_symlink prefix/"R.framework/Resources/man1/Rscript.1"
+
+    r_home = prefix/"R.framework/Resources"
+    # if this was built with a Homebrew gfortran, immunize to minor gcc version changes
+    if (r_home/"etc/Makeconf").read.include? Formula["gcc"].prefix
+      inreplace r_home/"etc/Makeconf", Formula["gcc"].prefix, Formula["gcc"].opt_prefix
+    end
+
+    # make Homebrew packages discoverable for R CMD INSTALL
+    inreplace r_home/"etc/Makeconf" do |s|
+      s.gsub!(/^CPPFLAGS =.*/, "\\0 -I#{HOMEBREW_PREFIX}/include")
+      s.gsub!(/^LDFLAGS =.*/, "\\0 -L#{HOMEBREW_PREFIX}/lib")
+      s.gsub!(/.LDFLAGS =.*/, "\\0 $(LDFLAGS)")
+    end
+
+    cd "src/nmath/standalone" do
+      system "make"
+      ENV.deparallelize # Serialized installs, please
+      system "make", "install"
+
+      lib.install_symlink Dir[prefix/"R.framework/Versions/[0-9]*/Resources/lib/libRmath.dylib"]
+      include.install_symlink Dir[prefix/"R.framework/Versions/[0-9]*/Resources/include/Rmath.h"]
+    end
+  end
+
+  def post_install
+    cellar_site_library = prefix/"R.framework/Resources/site-library"
+
+    ENV.delete "R_HOME" # Rscript prints garbage if R_HOME is set
+    installed_short_version = `#{bin}/Rscript -e 'cat(as.character(getRversion()[1,1:2]))'`.strip
+
+    site_library = HOMEBREW_PREFIX/"lib/R/#{installed_short_version}/site-library"
+    site_library.mkpath
+    cellar_site_library.unlink if cellar_site_library.exist?
+    ln_sf site_library, cellar_site_library
+  end
+
+  def caveats
+    <<-EOS.undent
+      To enable rJava support, run the following command:
```

Comment:
Assuming we're keeping the Java option, is this caveat text relevant when building without the Java option?